plugins/git/git.plugin.zsh

```diff
@@ -1,177 +1,188 @@
+
+alias gc='git commit -v'
+alias gc!='git commit -v --amend'
+alias gca='git commit -v -a'
+alias gca!='git commit -v -a --amend'
+alias gcan!='git commit -v -a -s --no-edit --amend'
+alias gcl='git config --list'
+alias gclean='git reset --hard && git clean -dfx'
+alias gcm='git checkout master'
+alias gcmsg='git commit -m'
+alias gco='git checkout'
+alias gcount='git shortlog -sn'
+compdef gcount=git
+alias gcp='git cherry-pick'
+alias gcs='git commit -S'
+
+alias gd='git diff'
+alias gdc='git diff --cached'
+alias gdt='git diff-tree --no-commit-id --name-only -r'
+gdv() { git diff -w "$@" | view - }
+compdef _git gdv=git-diff
+alias gdw='git diff --word-diff'
+
+alias gf='git fetch'
+alias gfa='git fetch --all --prune'
+function gfg() { git ls-files | grep $@ }
+compdef gfg=grep
+
+alias gg='git gui citool'
+alias gga='git gui citool --amend'
+ggf() {
+[[ "$#" != 1 ]] && b="$(current_branch)"
+git push --force origin "${b:=$1}"
+}
+compdef _git ggf=git-checkout
+ggl() {
+[[ "$#" != 1 ]] && b="$(current_branch)"
+git pull origin "${b:=$1}"
+}
+compdef _git ggl=git-checkout
+ggp() {
+[[ "$#" != 1 ]] && b="$(current_branch)"
+git push origin "${b:=$1}"
+}
+compdef _git ggp=git-checkout
+ggpnp() {
+ggl "$1" && ggp "$1"
+}
+compdef _git ggpnp=git-checkout
+ggu() {
+[[ "$#" != 1 ]] && b="$(current_branch)"
+git pull --rebase origin "${b:=$1}"
+}
+compdef _git ggu=git-checkout
+
 alias gignore='git update-index --assume-unchanged'
-alias gunignore='git update-index --no-assume-unchanged'
-# list temporarily ignored files
 alias gignored='git ls-files -v | grep "^[[:lower:]]"'
+alias git-svn-dcommit-push='git svn dcommit && git push github master:svntrunk'
+compdef git-svn-dcommit-push=git
+
+alias gk='\gitk --all --branches'
+compdef _git gk='gitk'
+alias gke='\gitk --all $(git log -g --pretty=format:%h)'
+compdef _git gke='gitk'
+
+alias gl='git pull'
+alias glg='git log --stat --color'
+alias glgp='git log --stat --color -p'
+alias glgg='git log --graph --color'
+alias glgga='git log --graph --decorate --all'
+alias glgm='git log --graph --max-count=10'
+alias glo='git log --oneline --decorate --color'
+alias glol="git log --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --abbrev-commit"
+alias glola="git log --graph --pretty=format:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue)<%an>%Creset' --abbrev-commit --all"
+alias glog='git log --oneline --decorate --color --graph'
+alias glp="_git_log_prettily"
+compdef _git glp=git-log
+
+alias gm='git merge'
+alias gmom='git merge origin/master'
+alias gmt='git mergetool --no-prompt'
+alias gmtvim='git mergetool --no-prompt --tool=vimdiff'
+alias gmum='git merge upstream/master'
+
+alias gp='git push'
+alias gpd='git push --dry-run'
+alias gpf='git push --force'
+alias gpoat='git push origin --all && git push origin --tags'
+compdef _git gpoat=git-push
+alias gpu='git push upstream'
+
+alias gr='git remote'
```

Comment:
This may satisfy a need for consistency, but I am not sure it really helps, and it would make those aliases one character longer, so I do not think this is the right choice.